tests/integration/helpers/sort-by-test.js

```diff
@@ -44,7 +44,7 @@ module('Integration | Helper | {{sort-by}}', function(hooks) {
       {{~/each~}}
     `);
 
-    assert.equal(find('*').textContent.trim(), 'aAAabccb', 'sorts multiletter words');
+    assert.equal(find('*').textContent.trim(), 'AaaAbccB', 'sorts multiletter words');
```

Comment:
We aren't splitting the strings up by individual char, so I actually think this is now "more" right.